packages/sanity/src/structure/panes/document/documentPanel/header/DocumentPanelHeader.tsx

```diff
@@ -37,6 +39,33 @@ export interface DocumentPanelHeaderProps {
   menuItems: PaneMenuItem[]
 }
 
+const HorizontalScroller = styled(Card)((props) => {
+  const theme = getTheme_v2(props.theme)
+
+  return css`
+    scrollbar-width: none;
+    z-index: 1;
+    position: relative;
+    > div {
+      &::-webkit-scrollbar {
+        width: 0;
+        height: 0;
+      }
+    }
+
+    &::after {
+      content: '';
+      display: block;
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      width: ${theme.space[3]}px;
+      background: linear-gradient(to right, ${rgba(theme.color.bg, 0)}, var(--card-bg-color));
+    }
+  `
+})
```

Comment:
Wonder if it might be worth moving this to the components-ui section? 🤔  feels like it might be useful?

Reply:
Fair, I'm just thinking that since we are introducing it in such a visible part of the code that the likelihood of us reusing this pattern is likely high. I don't feel strongly about this though :)